test/e2e_node/runner/run_e2e.go

```diff
@@ -122,41 +137,102 @@ func main() {
 	}
 }
 
-func WaitForUser() {
-	scanner := bufio.NewScanner(os.Stdin)
-	fmt.Printf("Enter \"y\" to run tests\n")
-	for scanner.Scan() {
-		if strings.ToUpper(scanner.Text()) != "Y\n" {
-			break
+func testHost(host string) *TestResult {
+	return RunTestsAgainstHost(*project, *zone, host, kubeRoot)
+}
+
+func testImage(image string) *TestResult {
+	err := createInstance(image)
+	defer deleteInstance(image)
+	if err != nil {
+		return &TestResult{
+			Err: fmt.Errorf("Unable to create gce instance for image %s.  %v", image, err),
 		}
-		fmt.Printf("Enter \"y\" to run tests\n")
 	}
-	u.Done()
+	result := RunTestsAgainstHost(*project, *zone, imageToInstanceName(image), kubeRoot)
+	return result
 }
 
-type TestResult struct {
-	fullhost                string
-	err                     error
-	testCombinedOutput      string
-	etcdCombinedOutput      string
-	apiServerCombinedOutput string
-	kubeletCombinedOutput   string
+func createInstance(image string) error {
+	name := imageToInstanceName(image)
+	i := &compute.Instance{
+		Name:        name,
+		MachineType: machineType(),
+		NetworkInterfaces: []*compute.NetworkInterface{
+			{
+				AccessConfigs: []*compute.AccessConfig{
+					{
+						Type: "ONE_TO_ONE_NAT",
+						Name: "External NAT",
+					},
+				}},
+		},
+		Disks: []*compute.AttachedDisk{
+			{
+				AutoDelete: true,
+				Boot:       true,
+				Type:       "PERSISTENT",
```

Comment:
I doubt we would, but I also don't think we are saving much by shrinking it to save 6-8GB.  The CPU and ram are more costly.  The requested size cannot be smaller than the image size, so shrinking it would make creating images more error prone since we would have to always change the defaults.